### DISCUSSION TITLE
Adding hawku anti-smooting filter

### DISCUSSION THREAD

**Discussion by ArchTemp** [OpenTabletDriver/OpenTabletDriver#1237] [Ideas]
add hawku anti-smooting filter such as hawku driver

**Comment by InfinityGhost** [comment]
Install the TabletDriverFilters plugin. It provides exactly what you're looking for.

**Comment by X9VoiD** [comment]
Hawku anti-smoothing is useless. Use [Reconstructor](https://github.com/X9VoiD/VoiDPlugins/wiki/Reconstructor) instead.
